rustls-platform-verifier/src/lib.rs

```diff
@@ -65,6 +65,23 @@ pub fn tls_config() -> ClientConfig {
         .with_no_client_auth()
 }
 
+/// Attempts to construct a `rustls` configuration that verifies TLS certificates in the best way
+/// for the underlying OS platform, using the provided
+/// [`CryptoProvider`][rustls::crypto::CryptoProvider].
+///
```

Comment:
Suggestion: It might be nice to say something like "See `tls_config` for an example and other details" here since that also mentions how to configure other parts of `ClientConfig`.